src/Snowflake/ConnectionExceptionResolver.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\Db\Import\Snowflake;
+
+use Keboola\Db\Import;
+
+class ConnectionExceptionResolver
+{
+    public static function resolveException(\Throwable $e): void
```

Comment:
Něco podobného jsme řešili u teradata ex a přijde mi srozumitelnější a snadněji otestovatelný když ten resolver ty exception vrací místo toho aby je rovnou vyhazoval.  https://github.com/keboola/ex-teradata/blob/eea662574215bde75a4d217849cb0f5532cb1c4a/src/ExceptionHandler.php#L61

Reply:
už tady jsem se trochu ztratil jestli `processErrorException` vyhodí vždy exception nebo jenom když splní tu podmínku

Reply:
Jo přesně tohle jsem hledal, protože mi to už @tomasfejfar. Akorát jsem to nenašel. Taky mi to přijde lepší. Předělám to!

Reply:
Done. 

To použití &matches v tý nový vyjímce je fakt dobrej nápad!!